config/ember-try.js

```diff
@@ -1,96 +1,111 @@
 "use strict";
 
-const getChannelURL = require("ember-source-channel-url");
+const getChannelURL = require('ember-source-channel-url');
+const { embroiderSafe, embroiderOptimized } = require('@embroider/test-setup');
 
-module.exports = function() {
-  return Promise.all([
-    getChannelURL("release"),
-    getChannelURL("beta"),
-    getChannelURL("canary")
-  ]).then(urls => {
-    return {
-      useYarn: true,
-      scenarios: [
-        {
-          name: "ember-lts-3.4",
-          npm: {
-            devDependencies: {
-              "ember-source": "~3.4.0",
-              "ember-data": "~3.4.0"
-            }
-          }
+module.exports = async function () {
+  return {
+    useYarn: true,
+    scenarios: [
+      {
+        name: 'ember-lts-3.20',
+        npm: {
+          devDependencies: {
+            'ember-source': '~3.20.7',
+            'ember-data': '~3.20.5'
+          },
         },
-        {
-          name: "ember-lts-3.8",
-          npm: {
-            devDependencies: {
-              "ember-source": "~3.8.0",
-              "ember-data": "~3.8.0"
-            }
-          }
+      },
+      {
+        name: 'ember-lts-3.24',
+        npm: {
+          devDependencies: {
+            'ember-source': '~3.24.3',
+            'ember-data': '~3.24.2'
+          },
         },
-        {
-          name: "ember-lts-3.12",
-          npm: {
-            devDependencies: {
-              "ember-source": "~3.12.0",
-              "ember-data": "~3.12.0"
-            },
-            resolutions: {
-              "ember-data": "~3.12.0"
-            }
+      },
+      {
+        name: "fastboot-1.2",
+        npm: {
+          dependencies: {
+            fastboot: "~1.2.1"
```

Comment:
this actually was not working as `fastboot` needs to be in `devDependencies` as can be found in `package.json`